### ISSUE TITLE
postgres_fdw does not work with Greenplum when connect to standalone PostgreSQL

### ISSUE THREAD

**Issue by yydzero** [greenplum-db/gpdb#10043]
### Greenplum version or build
Greenplum 6
PostgreSQL 9-12

Sounds like postgres_fdw in Greenplum uses Greenplum's changed libpq whose version number is not recognized by remote PostgreSQL.

```
SELECT * from foreign_table ;                                                                                   
ERROR:  could not connect to server "foreign_server"
DETAIL:  FATAL:  unsupported frontend protocol 28675.0: server supports 2.0 to 3.0
```

**Comment by yydzero** [comment]
connectDBStart:
```
#ifndef FRONTEND
	// GPDB uses the high bits of the major version to indicate special internal communications
	conn->pversion = GPDB_INTERNAL_PROTOCOL(3, 0);
#else
	conn->pversion = PG_PROTOCOL(3, 0);
#endif
```

**Comment by hlinnaka** [comment]
There was some discussion on this at https://github.com/greenplum-db/gpdb/pull/9462. I'm not sure what the status is.

**Comment by adam8157** [comment]
Fixed by #9462
